controllers/news-controller.js

```diff
@@ -35,4 +35,12 @@ exports.getComments = (req, res, next) => {
         res.status(200).send(comments)
     })
     .catch(next)
-};
\ No newline at end of file
+};
+
+exports.deleteComment = (req, res, next) => {
+    const id = req.params.comment_id;
+    deleteComment(id).then((comment) => {
+        res.status(204).send("204 - no content")
```

Comment:
`204` codes cannot have any content so the `.send` is a little redundant here!

you can use `.sendStatus(204)` to avoid having to use a `.send`